main/templates/subsidy/create.html

```diff
@@ -102,9 +102,9 @@
   let compare = () => {
     let startValue = (new Date(startInput.value)).getTime();
     let endValue = (new Date(endInput.value)).getTime();
-    let presentationValue = (new Date(presentationInput.value));
-    let paymentValue = (new Date(paymentInput.value));
-    let provisionalValue = (new Date(provisionalInput.value));
+    let presentationValue = (new Date(presentationInput.value)).getTime();
+    let paymentValue = (new Date(paymentInput.value)).getTime();
+    let provisionalValue = (new Date(provisionalInput.value)).getTime();
 
```

Comment:
The only thing I think you are not taking into account is the final resolution date. You should validate that the final resolution date must be after the provisional resolution date.